source/class/qx/ui/core/scroll/MRoll.js

```diff
@@ -24,6 +24,16 @@
  * attach the listener yourself.
  */
 qx.Mixin.define("qx.ui.core.scroll.MRoll", {
+  /**
+   * A possibility to scroll an area by dragging the area. If the property has false value then drag and drog works and if true doesn't.
+   */
+  properties: {
+    onlyRoll: {
+      init: false,
+      check: "Boolean"
+    }
+  },
+
```

Comment:
`onlyRoll` doesn't seem, to me, to indicate what this feature is allowing and preventing. What do you think of this rename of the property (and updated comment), instead?

```
  /**
   * Allow scrolling by dragging the area. CAUTION: Enabling this property disallows the capability of drag&drop.
   */
  properties: {
    scrollByDrag: {
      init: false,
      check: "Boolean"
    }
  },
```